contracts/governance/BaseWeightedMultisig.sol

```diff
@@ -275,7 +278,7 @@ abstract contract BaseWeightedMultisig is IBaseWeightedMultisig {
      * @notice Gets the specific storage location for preventing upgrade collisions
      * @return slot containing the BaseWeightedMultisigStorage struct
      */
-    function _baseWeightedMultisigStorage() private pure returns (BaseWeightedMultisigStorage storage slot) {
+    function _baseWeightedMultisigStorage() internal pure returns (BaseWeightedMultisigStorage storage slot) {
```

Comment:
this change makes it easier to modify it during upgrades within the gateway contract _setup if needed